package.json

```diff
@@ -40,6 +40,7 @@
     "react-dom": "18.2.0",
     "react-dropzone": "^14.3.5",
     "react-json-view": "1.21.3",
+    "react-tailwindcss-select": "^1.8.5",
```

Comment:
we were using this package before and made OS contributions to it as well. we could replicate some of its behavior via combobox but that requires a lot of work so opting in for this one again.